src/nifax/control/controller/ProductController.java

```diff
@@ -0,0 +1,49 @@
+package nifax.control.controller;
+
+import java.util.List;
+import nifax.control.model.CategoryProduct;
+import nifax.control.model.Product;
+import nifax.control.model.modeler.ModelOperation;
+import nifax.control.model.modeler.operation.IProductOperation;
+import nifax.control.model.modeler.operation.IQueries;
+
+/**
+ *
+ * @author NB
+ */
+public class ProductController extends ModelOperation
```

Comment:
[CC] If the class is in controller package, the "Controller" name is not neccesary on the class name.